src/web-animations-next-animation.js

```diff
@@ -44,8 +59,21 @@
         this._animation = scope.newUnderlyingAnimationForGroup(this.effect);
         scope.bindAnimationForGroup(this);
       }
-
-      // FIXME: move existing currentTime/startTime/playState to new animation
+      if (hasUnderlying) {
+        if (oldPlaybackRate != 1) {
+          this.playbackRate = oldPlaybackRate;
+        }
+        if (oldPaused) {
+          this.pause();
+        }
+        if (oldStartTime !== null) {
+          this.startTime = oldStartTime;
+        } else if (oldCurrentTime !== null) {
+          this.currentTime = oldCurrentTime;
```

Comment:
> We should prefer setting the current time over the start time.
> 
> Otherwise after a.play(); a.startTime = x; the currentTime will still be pending

In an earlier revision you said "I think we should be setting the start time, not the current time. Otherwise the start time will still be unresolved."

There might be a mismatch between the Chrome impl and the polyfill impl of Animation timing setters.

In the polyfill, the startTime setter immediately updates the currentTime. The currentTime setter only updates startTime if the existing startTime it is non-null.

In the native impl is looks as if the startTime setter doesn't set currentTime (unless the animation is held). The currentTime setter updates the startTime immediately.